### ISSUE TITLE
--root option broken

### ISSUE THREAD

**Issue by anisse** [jakubpawlowicz/clean-css#734]
--root option is broken for import inlining since commit f22851cb5a4f21ecf2189a998757724c8e910a93 "Fixes #400 - CleanCSS#minify accepts an array of filenames."

In particular for imports that don't start with "/".

It somehow uses dirname(options.root) for options.relativeTo, which makes context.relativeTo pointing to the parent directory.

Here is the bisection script used with "git bisect run":
```shell
#!/bin/sh
echo "@import url(footer.css);" | bin/cleancss --root /home/anisse/website/static/css/ > /dev/null
```

There should be a footer.css in the website's static/css directory.

**Comment by jakubpawlowicz** [comment]
That's clearly wrong! Let's fix it.

**Comment by anisse** [comment]
Just to provide some context, this broke my pelican install, which uses webassets which uses cleancss. I guess not many people mix python and node projects like this, which is why there was no report before this one.

**Comment by jakubpawlowicz** [comment]
Fix already on branch 3.4, will be released at some point soon.
